### PR TITLE
client: handle sellstring

### LINKED ISSUE #1412: intl: deal with sellString
https://github.com/decred/dcrdex/blob/e1218a26c8c91abe77ef9a6cf92f801db1399183/client/core/trade.go#L2544-L2549

When dealing with the `core` `sellString`, we should avoid using the string in translation subject templates that do `%sing` where "selling"/"buying" conjugations are needed because that's not workable for most languages.

https://github.com/decred/dcrdex/blob/17a96d1024895224dce835dc9db5e802f9ba1ba5/client/webserver/site/src/js/orderutil.js#L39

### PR DESCRIPTION
**Pull request by ukane-philemon**
This diff improves `sellstring(ord :Order)` in `orderutil.js` and handles `buy || Sell` && `Buying || Selling`string for `core` notifications. Closes #1412 && #1770

### CODE REVIEW

**Review by JoeGruffins — APPROVED** [review]

File: client/webserver/site/src/js/locales.ts [hunk 1]
```diff
@@ -210,7 +214,7 @@ export const zhCN: Locale = {
   [ID_API_ERROR]: '接口错误',
   [ID_ADD]: '加',
   [ID_CREATE]: '创建',
-  [ID_WALLET_READY]: 'Choose Wallet', // xxx translate
+  [ID_WALLET_READY]: 'Ready', // xxx translate
   [ID_SETUP_WALLET]: 'Setup' // xxx translate
```
Comment (buck54321) [inline client/webserver/site/src/js/locales.ts:218]: Can we delete these `xxx translate` entries? We already fall back to English, and having these entries inhibits our ability to generate a list of missing translations.

File: client/webserver/site/src/js/orderutil.ts [hunk 2]
```diff
@@ -44,7 +44,12 @@ export const Taker = 1
...
 export const RateEncodingFactor = 1e8
 
-export function sellString (ord: Order) { return ord.sell ? 'sell' : 'buy' }
+export function sellString (ord: Order) {
+  const key = ord.sell ? intl.ID_SELL : intl.ID_BUY
+  const lang = document.documentElement.lang.toLowerCase()
+  return intl.prep(key).toLocaleLowerCase(lang)
+}
+
 export function typeString (ord: Order) { return ord.type === Limit ? (ord.tif === ImmediateTiF ? 'limit (i)' : 'limit') : 'market' }
```
Comment (chappjc) [inline client/webserver/site/src/js/orderutil.ts:53]: :sob: It never ends.  Next I guess...

**Review by chappjc — APPROVED** [review]
Looks good, but let's see about a change to deal with the `RETRANSLATE` issue better.

https://github.com/decred/dcrdex/commit/a7a3196494e10c8dd0fdb7ca9143c747e5e661df from https://github.com/decred/dcrdex/pull/1835

File: client/core/locale_ntfn.go [hunk 3]
```diff
@@ -380,10 +394,15 @@ var ptBR = map[Topic]*translation{
...
+	// [qty, ticker, rate string, token], RETRANSLATE.
+	TopicBuyOrderPlaced: {
 		subject:  "Ordem Colocada",
-		template: "%sing %s %s, valor = %s (%s)",
+		template: "Buying %s %s, valor = %s (%s)",
+	},
+	// [qty, ticker, rate string, token], RETRANSLATE.
+	TopicSellOrderPlaced: {
+		subject:  "Ordem Colocada",
+		template: "Selling %s %s, valor = %s (%s)",
```
[the remark this answers is not in the document]
Comment (buck54321) [reply client/core/locale_ntfn.go:405]: Good plan

[the remark this answers is not in the document]
Comment (chappjc) [reply client/core/locale_ntfn.go:405]: k, so we'll get https://github.com/decred/dcrdex/pull/1835 in first, then we can update these with `stale: true,` and remove the `RETRANSLATE` comments.

[the remark this answers is not in the document]
Comment (ukane-philemon) [reply client/core/locale_ntfn.go:405]: Alright.

File: client/webserver/site/src/js/orderutil.ts (hunk 2 above)
Comment (ukane-philemon) [reply client/webserver/site/src/js/orderutil.ts:53]: Yeah. I could look into other static strings like typeString in another PR.

**Pull request merged**